adoc/Release-Notes.adoc

```diff
@@ -31,6 +31,112 @@ endif::[]
+** cf-mysql: 36.15.0
+** cf-routing: 0.184.0
+** cf-sle12: 1.75.11
+** cf-smoke-tests: 40.0.44
+** cf-syslog-drain: 8.1
+** cf-usb: 1.0.1
+** cflinuxfs2: 1.281.0
+** cflinuxfs3: 0.81.0
+** credhub: 2.1.2
+** diego: 2.25.0
+** eirini: 0.0.4
+** garden-runc: 1.17.2
+** groot-btrfs: 1.0.4
+** kubectl: 1.9.6
+** loggregator: 104.4
+** loggregator-agent: 3.2
+** nats: 26
+** nfs-volume: 1.7.6
+** postgres-release: 26
+** scf-helper: 1.0.2
+** cf-acceptance-tests: 
+** statsd-injector: 1.5.0
+** uaa: 68.0
+* Buildpacks:
+** binary-buildpack: 1.0.32
+** dotnet-core-buildpack: 2.2.10
+** go-buildpack: 1.8.36
+** java-buildpack: 4.19.1 
+** nginx-buildpack: 1.0.11
+** nodejs-buildpack: 1.6.49
+** php-buildpack: 4.3.75
+** python-buildpack: 1.6.32
+** ruby-buildpack: 1.7.38
+** staticfile-buildpack: 1.4.42
+
+[id='sec.1_4.issue']
+==== Known Issues
+
+* The instructions for enabling Eirini can be found at https://github.com/SUSE/scf/wiki/Eirini.
+
+* Resuming a past practice, with {product} 1.4, use the complete command: `helm upgrade --force --recreate-pods` for an upgrade. This will reintroduce downtime for apps but without `--recreate-pods`, multiple versions of statefulsets may co-exist which can cause incompatibilities between depedent statefulsets, and result in a broken upgrade. This applies to Stratos pods as well.
+
+* With the introduction of feature flags, setting `sizing.<role>.count` to enable/disable a feature is no longer supported. You must explicitly set `enable.<feature>` to `true` or `false` to enable/disable a feature. As an example, if you had enabled credhub or autoscaler in {product} 1.3.1, then you must add `enable.credhub=true` or `enable.autoscaler=true` during the `helm upgrade`. If you had previously set `sizing.<role>.count` to `1` you can remove that as the new minimum setting is `1`. Conversely, if you had disabled a feature in {product} 1.3.1, you should remove the corresponding sizing setting and, instead, explicitly set `enable.<feature>=false` during the upgrade. If you would like to deploy more than `1` instance of an optional role, you would need to use an appropriate value for `sizing.<role>.count` in addition to using the feature flag.
+
+* If autoscaler was enabled in {product} 1.3.1, you must specify `sizing.autoscaler_postgres.disk_sizes.postgres_data=100` during the helm upgrade to avoid upgrade errors. Alternatively, you can disable the autoscaler before the upgrade and re-enable after the upgrade is finished. Without any of these workarounds, the upgrade would fail with `Error: UPGRADE FAILED: StatefulSet.apps "autoscaler-postgres" is invalid` message.
+
+* If you are using the NGINX Ingress Controller and seeing `Request Entity Too Large` errors, you should bump up the ingress proxy body size to an appropriate value by setting the `ingress.annotations` key in helm chart values as in the following:
+
+```
+  ingress:
+     annotations:
+       nginx.ingress.kubernetes.io/proxy-body-size: 64m
+```
```

Comment:
This screen is formatted like Markdown, and won't work in AsciiDoc. It should look like:
 
```
+
[source]
----
  ingress:
     annotations:
       nginx.ingress.kubernetes.io/proxy-body-size: 64m
----
```

Note the `+` in the first line (i.e. replacement for empty line 126), that will keep the screen within the list indentation.

Reply:
Done in https://github.com/SUSE/cap-release-notes/pull/61